docs/developer_docs.md

```diff
@@ -43,6 +44,12 @@ This plugin supports sending text messages to the Virtual Agent through Mattermo
     - [Pre-defined Conversation Flows/Topics](https://docs.servicenow.com/bundle/quebec-now-intelligence/page/administer/virtual-agent/reference/prebuilt-topics-ITSM.html)
     - [Additional Plugins](https://docs.servicenow.com/bundle/sandiego-servicenow-platform/page/administer/virtual-agent/reference/additional-va-plugins.html)
 
+- **File Upload**
+
+    For sending file attachments to the Virtual Agent or the Live Agent we need to send a public link of the file in the request from where Virtual Agent can download the file. Virtual Agent does not support sending file attachments from any source, we have to specify the trusted domains ourselves so that the Virtual Agent knows that these domains can be trusted.
```

Comment:
```suggestion
    For sending file attachments to the Virtual Agent or the Live Agent we need to send a public link of the file from where the Virtual Agent can download the file. Virtual Agent does not support sending file attachments from any source, we have to specify the trusted domains ourselves so that the Virtual Agent knows that these domains can be trusted.
```